### DISCUSSION TITLE
2023/04/01/leetcode%E5%88%B7%E9%A2%98%E8%AE%B0%E5%BD%957-%E6%89%BE%E5%88%B0%E4%BA%8C%E5%8F%89%E6%A0%91%E4%B8%AD%E6%9C%80%E8%BF%91%E7%9A%84%E5%8F%B3%E4%BE%A7%E8%8A%82%E7%82%B9/

### DISCUSSION THREAD

**Discussion by giscus** [Zhuanjier/blog#15] [Announcements]
# 2023/04/01/leetcode%E5%88%B7%E9%A2%98%E8%AE%B0%E5%BD%957-%E6%89%BE%E5%88%B0%E4%BA%8C%E5%8F%89%E6%A0%91%E4%B8%AD%E6%9C%80%E8%BF%91%E7%9A%84%E5%8F%B3%E4%BE%A7%E8%8A%82%E7%82%B9/

¶找到二叉树中最近的右侧节点  原题传送门  给定一棵二叉树的根节点 root 和树中的一个节点 u ，返回与 u 所在层中距离最近的右侧节点，当 u 是所在层中最右侧的节点，返回 null 。 ¶示例 ¶示例1  输入：root = [1,2,3,null,4,5,6], u = 4 输出：5 解释：节点 4 所在层中，最近的右侧节点是节点 5。 ¶示例2  输入：root = [3,null,

https://blog.zhuanjie.ltd/2023/04/01/leetcode%E5%88%B7%E9%A2%98%E8%AE%B0%E5%BD%957-%E6%89%BE%E5%88%B0%E4%BA%8C%E5%8F%89%E6%A0%91%E4%B8%AD%E6%9C%80%E8%BF%91%E7%9A%84%E5%8F%B3%E4%BE%A7%E8%8A%82%E7%82%B9/

**Comment by RandallAnjie** [comment]
空指针： `nullptr` ；
空值： `null`
